src/ParkChecklist/ParkChecklist.js

```diff
@@ -98,116 +100,130 @@ const ParkChecklist = ({ parks, setParks, apiLink, selectedState, setSelectedSta
     <div className="park-checklist-container">
       {loading ? <p className="loading-message">Loading Parks...</p> : (
         <section>
-      <div className="below-header-box">
-        <div className="state-selector">
-          <select onChange={handleStateChange} value={selectedState}>
-            <option value="">Select a State</option>
-            {states.map((stateCode) => (
-              <option key={stateCode} value={stateCode}>
-                {stateCode}
-              </option>
-            ))}
-          </select>
-          <CustomButton onClick={handleGoClick}>Go!</CustomButton>
-        </div>
-      </div>
-      <div className="sorting-options">
-        <div className="sort-by-state">
-          <StateSort
-            states={states}
-            selectedStateParks={selectedStateParks}
-            handleStateCheckboxChange={handleStateCheckboxChange}
-          />
-        </div>
-        <DesignationSort
-          getUniqueDesignations={getUniqueDesignations}
-          selectedDesignations={selectedDesignations}
-          handleDesignationCheckboxChange={handleDesignationCheckboxChange}
-        />
-      </div>
-      <h2>{`${totalChecked}/${parks.length} parks visited`}</h2>
-      <h2>Parks Checklist</h2>
-      <ul className="checkbox-list">
-        {selectedFilters.parks.length > 0 ? (
-          selectedFilters.parks
-            .filter(
-              (park) =>
-                selectedFilters.designations.length === 0 ||
-                selectedFilters.designations.some(
-                  (designation) =>
-                    park.attributes.designation === designation
-                )
-            )
-            .map((park) => (
-              <li key={park.id}>
-                <input
-                  type="checkbox"
-                  id={park.id}
-                  checked={checkedItems.includes(park.id)}
-                  onChange={() => handleCheckboxChange(park.id)}
+          <div className="below-header-box">
+            <div className="state-selector">
+              <select onChange={handleStateChange} value={selectedState}>
+                <option value="">Select a State</option>
+                {states.map((stateCode) => (
+                  <option key={stateCode} value={stateCode}>
+                    {stateCode}
+                  </option>
+                ))}
+              </select>
+              <CustomButton onClick={handleGoClick}>Go!</CustomButton>
+            </div>
+            <div className="sort-buttons">
+              <CustomButton onClick={() => setShowStateModal(true)}>State Sort</CustomButton>
```

Comment:
Having these triggered on buttons will make the page much friendlier up front, well done!